leap_c/examples/hvac/env.py

```diff
@@ -561,7 +561,26 @@ def reset(
         else:
             min_start_idx = 0
             max_start_idx = len(self.data) - self.N_forecast - self.max_steps + 1
-            self.idx = self.np_random.integers(low=min_start_idx, high=max_start_idx + 1)
+
+            self.idx = min_start_idx
+
+            n_max = 1000
+            counter = 0
+            date_valid = False
+            while not date_valid and counter < n_max:
+                idx = self.np_random.integers(low=min_start_idx, high=max_start_idx + 1)
+                date = self.data.index[idx]
+                if date.month in [1, 2, 3, 10, 11, 12]:
```

Comment:
Is the heating period really so restrictive? E.g., I would also count the April in Germany, and depending on the data, e.g., countries more to the north, it might include even more months?